hbase_sql/src/main/java/com/splicemachine/derby/stream/function/BulkImportFunction.java

```diff
@@ -83,4 +106,18 @@ public void readExternal(ObjectInput in) throws IOException, ClassNotFoundExcept
         bulkImportDirectory = (String) in.readObject();
         SpliceSpark.setupSpliceStaticComponents();
     }
+
+    private void init(Iterator<BulkImportPartition> importPartitions) throws Exception {
+        partitionMap = new HashMap<>();
+        while (importPartitions.hasNext()) {
+            BulkImportPartition partition = importPartitions.next();
+            Long conglom = partition.getConglomerateId();
+            List<BulkImportPartition> partitionList = partitionMap.get(conglom);
+            if (partitionList == null) {
+                partitionList = Lists.newArrayList();
+                partitionMap.put(conglom, partitionList);
+            }
```

Comment:
The code makes the assumption that there is only one unique conglomerate possible in the importPartitions list, because partitionMap.put is only called once.  Could index conglomerates also be in the list?  Either we should add a sanity check to make sure each call to getConglomerateId() returns the same conglomerate (if we only expect there to ever be just one), or we should make a call to partitionMap.put for each unique conglomerate.

Reply:
This code add one entry to partitionMap for each conglomerate, including base table and indexes.

Reply:
I misread it, didn't see the call to partitionMap.get.  Looks good.